src/translations/fr.json

```diff
@@ -69,8 +69,8 @@
     "edit": "Modifier",
     "createNewContingencyList": "Créer une liste d'aléas",
     "FORM": "Formulaire",
-    "editContingencyList": "Editer la liste d'aléas",
-    "editFilter": "Editer le filtre",
+    "editContingencyList": "Éditer la liste d'aléas",
+    "editFilter": "Éditer le filtre",
```

Comment:
why those changes ?

Reply:
Because it is how french must be written, it took me 5s, and it can't cause any bug.

Reply:
Ok to keep it. But for lisibility of PR, tracability of the code, it's always harmful to change things that have nothing to do with the PR's subject